tests/new_api/test_device.py

```diff
@@ -842,32 +865,27 @@ def test_derivatives_tape_batch(self, phi, execute_and_derivatives, batch_obs):
         ops = [qml.Hadamard(0), qml.IsingXX(phi, wires=(0, 1))]
         qs2 = QuantumScript(ops, [qml.expval(qml.prod(qml.Z(0), qml.Z(1)))], trainable_params=[0])
 
+        expected_device = DefaultQubit(wires=4, max_workers=1)
+
         if execute_and_derivatives:
             results, jacs = device.execute_and_compute_derivatives((qs1, qs2))
+
+            expected, expected_jac = expected_device.execute_and_compute_derivatives((qs1, qs2))
         else:
             results = device.execute((qs1, qs2))
             jacs = device.compute_derivatives((qs1, qs2))
 
-        # Assert results
-        expected1 = (-np.sin(phi) - 1, 3 * np.cos(phi))
-        x1 = np.cos(phi / 2) ** 2 / 2
-        x2 = np.sin(phi / 2) ** 2 / 2
-        expected2 = sum([x1, -x2, -x1, x2])  # zero
-        expected = (expected1, expected2)
+            expected = expected_device.execute((qs1, qs2))
+            expected_jac = expected_device.compute_derivatives((qs1, qs2))
```

Comment:
Using `default.qubit` and ground truth to give more flexibility to the test

Reply:
This makes sense. But we are pushing to reduce the number of tests comparing directly against `default.qubit`.

Reply:
Ok. Do you suggest going back to the previous version or keeping these implementations? :thinking:

Reply:
If we are not losing much, it is better to go back to a `default.qubit`-independent implementation.

Reply:
Ok, I will try to revert these changes. :smile: